src/AMSlib/wf/basedb.hpp

```diff
@@ -693,28 +692,29 @@ class RedisDB : public BaseDB<TypeValue>
 
 /**
   * @brief AMS represents the header as follows:
-  * The header is 12 bytes long:
-  *   - 1 byte is the size of the header (here 12). Limit max: 255
+  * The header is 16 bytes long:
+  *   - 1 byte is the size of the header (here 16). Limit max: 255
   *   - 1 byte is the precision (4 for float, 8 for double). Limit max: 255
   *   - 2 bytes are the MPI rank (0 if AMS is not running with MPI). Limit max: 65535
   *   - 4 bytes are the number of elements in the message. Limit max: 2^32 - 1
   *   - 2 bytes are the input dimension. Limit max: 65535
   *   - 2 bytes are the output dimension. Limit max: 65535
-  * 
-  * |__Header__|__Datatype__|___Rank___|__#elems__|___InDim___|___OutDim___|...real data...|
-  * ^          ^            ^          ^          ^           ^            ^               ^
-  * |  Byte 1  |   Byte 2   | Byte 3-4 | Byte 4-8 | Byte 8-10 | Byte 10-12 |   Byte 12-X   |
+  *   - 4 bytes for padding. Limit max: 2^32 - 1
```

Comment:
Did we make the respective changes on the python side, to read in messages of header size 16?

Reply:
No, that's already there (cf. https://github.com/LLNL/AMS/pull/30/commits/68c36448a6b4139cf806e5e9da06a5a8153f38e3)